README.md

```diff
@@ -9,6 +9,12 @@ Sets up an API with data from [Serviceplatformen Organisation](https://digitalis
 * [Symfony](https://symfony.com)
 * [API Platform](https://api-platform.com/)
 
+## Disclaimer
+
+The currently suggested setup exposes the API via docker networks, see
+[Using the API from another docker compose setup](#using-the-api-from-another-docker-compose-setup).
+__It is very important that API is not exposed to the public.__
+
```

Comment:
Then why do we do it, i.e. suggest it? What is it we can't do if we don't expose the API to the public (by default)?

Reword to a more general warning that the API has no authentication methods and should not be exposed on the web.

Reply:
Reworded to a more general warning.